cds/modules/fixtures/cli.py

```diff
@@ -247,6 +247,11 @@ def page_data(page):
              description='FAQ',
              content=page_data('faq.html'),
              template_name='invenio_pages/dynamic.html'),
+        Page(url='/guide/search',
+             title='Search guide',
+             description='Search guide',
+             content=page_data('guide/search.html'),
```

Comment:
@costaflavio sorry but it is `guides/search.html`, and not `guide/search.html`.

![screen shot 2018-07-05 at 11 41 01](https://user-images.githubusercontent.com/2089455/42315705-699ab854-8048-11e8-8fb0-044e53a544cf.png)

And I would do the same for the url maybe, looks nicer?

Reply:
still missing fix for the guide->guides in the content path